examples/tv-app/linux/include/keypad-input/KeypadInputManager.cpp

```diff
@@ -51,7 +51,7 @@ EmberAfKeypadInputStatus KeypadInputManager::proxyKeypadInputRequest(EmberAfKeyp
     return EMBER_ZCL_KEYPAD_INPUT_STATUS_SUCCESS;
 }
 
-static void sendResponse(const char * responseName, uint8_t keypadInputStatus)
+static void sendResponse(const char * responseName, EmberAfKeypadInputStatus keypadInputStatus)
 {
     emberAfFillExternalBuffer((ZCL_CLUSTER_SPECIFIC_COMMAND | ZCL_FRAME_CONTROL_SERVER_TO_CLIENT), ZCL_KEYPAD_INPUT_CLUSTER_ID,
                               ZCL_SEND_KEY_RESPONSE_COMMAND_ID, "u", keypadInputStatus);
```

Comment:
This is not right, because "u" means "1-byte value" and we're now passing a 4-byte one.... https://github.com/project-chip/connectedhomeip/pull/7607 has the right fix.